pkg/server/public/tests/form_test.go

```diff
@@ -33,6 +33,11 @@ func (s *Suite) TestFormCreateGetList() {
 		return
 	}
 
+	selectOptions := []*types.SelectOption{
```

Comment:
is there a way in go to tag properties as required? if so, should the id of SelectOption not be required? since we're using typescript in the frontend, I think we would benefit of typesafety also in the backend

Reply:
In go, not really. It's possible to create any kind of struct without passing any fields. For types that are instantiated in the code, then we use a constructor with required arguments
```
NewFieldOption(id string, value..)
```
In this case, the type is deserialized, so there is no built-in way of doing this at compile time. The validator package would take care of that..